COLLABORATOR_GUIDE.md

```diff
@@ -1,28 +1,29 @@
 # Collaborator Guide
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**
-
-- [Issues and Pull Requests](#issues-and-pull-requests)
-- [Accepting changes](#accepting-changes)
-  - [Involving the TSC](#involving-the-tsc)
-- [Landing a PR](#landing-a-pr)
-  - [Landing a PR manually](#landing-a-pr-manually)
-    - [Landing a PR manually with several changes](#landing-a-pr-manually-with-several-changes)
-    - [I just made a mistake](#i-just-made-a-mistake)
-      - [I accidentally pushed a broken commit or incorrect commit to master](#i-accidentally-pushed-a-broken-commit-or-incorrect-commit-to-master)
-      - [I lost changes](#i-lost-changes)
-      - [I accidentally committed a broken change to master](#i-accidentally-committed-a-broken-change-to-master)
-- [video.js releases](#videojs-releases)
-  - [Getting dependencies](#getting-dependencies)
-    - [Install contrib](#install-contrib)
-    - [npm access](#npm-access)
-    - [GitHub personal access token](#github-personal-access-token)
-  - [Doing a release](#doing-a-release)
-- [Doc credit](#doc-credit)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Table of Contents
+
+* [Issues and Pull Requests](#issues-and-pull-requests)
+* [Accepting changes](#accepting-changes)
+  * [Involving the TSC](#involving-the-tsc)
+* [Landing a PR](#landing-a-pr)
+  * [Landing a PR manually](#landing-a-pr-manually)
+
```

Comment:
Looks like the TOC whitespace is a bug. I made a PR to fix it: https://github.com/BarryThePenguin/mdast-util-toc/pull/3